src/common/stylesheets/variables.scss

```diff
@@ -26,3 +26,6 @@ $primary-border-style: thin solid rgba(0, 0, 0, 0.05);
 
 // Responsive
 $responsive-width: 1024px;
+
+// Sizes
+$containerSize: 1024px;
```

Comment:
Does this mean we can dynamically adjust all the "root" containers? 
We've had some discussions as to where the 1024px was WAY too small on high res screens :P

Reply:
Yes sure, really just a max-width value which we can easily change and adjust